lowfi_simulation/src/main/native/include/MotorModel/LinearMotorModelSimulation.h

```diff
@@ -0,0 +1,40 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+#pragma once
+
+#include "MotorModel/MotorModelSimulation.h"
+
+namespace frc {
+namespace sim {
+namespace lowfi {
+
+class LinearMotorModelSimulation : public MotorModelSimulation {
+ public:
+  explicit LinearMotorModelSimulation(double maxSpeed);
+  ~LinearMotorModelSimulation();
+
+  virtual void Reset();
+  virtual void SetVoltage(double voltage);
+  virtual void Update(double elapsedTime);
+
+  virtual double GetPosition();
```

Comment:
Since the values returned are constants, these can be `virtual double GetPosition() const;` etc. Are derived classes expected to do computation in these functions? I would expect not since the Update() function exists.